CMakeLists.txt

```diff
@@ -148,7 +148,6 @@ function(configure_native_kernel kernel)
 endfunction()
 
 function(configure_wasm_kernel kernel)
-  set(XEUS_CPP_RESOURCE_DIR "/lib/clang/${CPPINTEROP_LLVM_VERSION_MAJOR}" PARENT_SCOPE)
```

Comment:
cc @SylvainCorlay , the `PARENT_SCOPE` variable was spoiling it for us. Just removing this (and also shifting it to a better location) should be the simple fix here.